### ISSUE TITLE
Hello, some questions

### ISSUE THREAD

**Issue by Gabrielk99** [pyannote/pyannote-audio#706]
Hey, good afternoon, or good morning or good evening, but have a good day.
Do you can answers some questions? please?
I will be grateful
1- I finetuned the SCD model, and validate. Well, the validate returned 99.5% of fscore, the loss of training step, is 8.5e⁻4.
2 - In my dataset, the model from validate step, aplyed in my test data, dont work very well like this
    ![Figure_3_prediction_change](https://user-images.githubusercontent.com/34044829/125321848-7d3d8680-e313-11eb-9b06-e4ce225d8a83.png)

3 - I tried aply the model in the development data and training data and the result is the same. So, i just thought, is my dataset very small? or my dataset have small set of many speakers?  
4 - Is the labels used to train the SCD model done using .rttm? meaning I don't need to worry about this step, right?
5 - This function peak(alpha=...) the params, I take it out of the params.yml from validate process?

**Comment by hbredin** [comment]
pyannote.audio won't support speaker change detection in future versions because it does not work that well anyway.

I'd recommend you check the [pyannote/segmentation](https://hf.co/pyannote/segmentation) model instead

**Comment by saumyaborwankar** [comment]
> pyannote.audio won't support speaker change detection in future versions because it does not work that well anyway.
> 
> I'd recommend you check the [pyannote/segmentation](https://hf.co/pyannote/segmentation) model instead

There is no code given for training segmentation model of pyannote. Could you help me with the retraining of segmentation model.

**Comment by hbredin** [comment]
Yes, there [is](https://github.com/pyannote/pyannote-audio#documentation).
